bootstrap/helpers/applications.php

```
@@ -29,6 +29,15 @@ function queue_application_deployment(Application $application, string $deployme
         $destination_id = $destination->id;
     }
 
+    // Use the application's pinned commit SHA when no specific commit was requested
+    // and this is not a PR or rollback deployment
+    if ($pull_request_id === 0 && ! $rollback) {
```

PHPMD (2.15.0)

[error] The variable $pull_request_id is not named in camelCase. (undefined)

(CamelCaseVariableName)